### PR TITLE
Add PR review process for 2021

### PR DESCRIPTION
**Pull request by wyc**
This process resulted from a series of meetings involving Brent Zundel, Wayne Chang, Manu Sporny, and Ivan Herman. Additional reviewers very welcome!

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Review by brentzundel — APPROVED** [review]

File: README.md
Comment (TallTed) [inline README.md:44]: I can't suggest what's needed here, because Github doesn't handle indented codefences properly.  Lines 38 through 44 need to be indented with 3 space characters, in order to nest that codeblock under item 2.

Comment (gkellogg) [reply README.md:44]: A definition list might be the way to go. It looks like GitHub doesn't support the extended [Markdown Definition List](https://www.markdownguide.org/extended-syntax/#definition-lists), at least not in comments. HTML in Markdown shouldn't be a problem.

**Review by kdenhartog — APPROVED** [review]
Looks good. Looking forward to keeping this work moving forward as well!

File: README.md
Comment (davidlehn) [inline README.md:61]:
```suggestion
- [v2] VC `@context` needs updating, possibly with security vocab modularized
```

File: README.md
Comment (msporny) [inline README.md:44]:
```suggestion
<dl>
  <dt>Editorial</dt>
  <dd>Mark with "editorial" tag, merge into branch "v1.1"</dd>
  <dt>Substantive</dt>
  <dd>Mark with "substantial" tag</dd>
  <dt>Bugfixes - merge into separate branch "v1.2". New Features stay around as an open PR.</dd>
</dl>
```

Per @gkellogg's comments, here's a `dl` suggestion.

Comment (TallTed) [reply README.md:44]: Opinions about mixing HTML with Markdown vary. I don't have strong feelings about it, and @msporny's suggestion seems fine, as far as it goes. My concern here is only that the definition list (or whatever) nest properly within/beneath step 2, rather than confusingly being on the same visual level as steps 2 and 3.

Comment (msporny) [reply README.md:44]: I agree on the formatting concerns, @TallTed -- we can rewrite the entire list in HTML if Markdown doesn't do the right thing. I hesitate to experiment w/ this in a PR. Once this is merged, I'll ensure it's properly formatted.

**Comment by msporny** [comment]
This is ready to merge based on the call today.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-06-14-vcwg#section4-1) on 2021-06-14

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.1. Add PR review process for 2021 (pr vc-data-model#774)

_See github pull request [#774](https://github.com/w3c/vc-data-model/pull/774)._




**Manu Sporny:** Wayne's PR, some suggestion were made in the comment, they have not been processed  
… pr 774 is ready to go in, unless there are objections on this call  
… this is the agreed upon process, and has been out for review - suggest to merge  

**David Chadwick:** clarify - what is version 1.2 (in PRs)  

**Brent Zundel:** quick editorial changes are set for version 1.1 and can go out a.s.a.p. - substantial changes, bugs, etc. are in version 1.2  

> *Ivan Herman:* +1 to DavidC

> *Michael Kupjetz:* everything else in version 2

**Brent Zundel:** any question?  

**Manu Sporny:** will merge pr 774 once the update has been made, comment added  

</details>

**Comment by brentzundel** [comment]
multiple positive reviews, feedback received and incorporated, merging

**Pull request merged**
